java/impl/aws/src/main/java/co/worklytics/psoxy/aws/ParameterStoreConfigService.java

```diff
@@ -70,7 +69,7 @@ public String load(String key) throws AwsServiceException {
                                     .build();
                                     GetParameterResponse parameterResponse = client.getParameter(parameterRequest);
                                     return parameterResponse.parameter().value();
-                                } catch (ParameterNotFoundException | ParameterVersionNotFoundException nfe) {
+                                } catch (SsmException ignore) {
```

Comment:
mmm ... I don't see why broaden this. You're now catching cases beyond the two "not found"s

Reply:
Yeah, had issues with errors related to policies when splitting into read/write and granting params per function.
I've split those and log in the non-general case

Reply:
To be clear: the issue is the nature of the fallback configuration approach, we try to read everything until we find it.
F.e. an environment variable, now with my policy restrictions, the error is not that is not found, but that there is no policy allowing to read that parameter, and that isn't an specific sub-classed `SsmException`.